ai/github_issue_handling/issue_clawer.py

```diff
@@ -0,0 +1,189 @@
+# -*- coding: utf-8 -*-
+
```

Comment:
This PR is only for server deployment. You may remove other files which exist in https://github.com/XinShuYang/antrea/pull/5/files

Reply:
Reminder